packages/chains/src/moonbeam/assets.ts

```diff
@@ -214,6 +217,30 @@ export const assets: SupportedAsset[] = [
     oracle: OracleTypes.CurveLpTokenPriceOracleNoRegistry,
     extraDocs: beamSwapStableDocs("nomad3pool", threePool),
   },
+  {
+    symbol: assetSymbols["STELLA_GLMR"],
+    underlying: STELLA_GLMR,
+    name: "Stella Swap STELLA/GLMR LP Token",
+    decimals: 18,
+    oracle: OracleTypes.UniswapLpTokenPriceOracle,
+    extraDocs: "",
```

Comment:
mind adding these as well? feel free to follow the structure of the `beamSwapDocs`, and create new entry for `stellaSwapDocs`